lib/ruby_zen/objects/method_object.rb

```diff
@@ -1,6 +1,7 @@
 module RubyZen
   class MethodObject
-    attr_reader :name, :owner, :parameters, :super_method, :return_object
+    attr_accessor :name, :owner, :parameters, :super_method, :return_object,
```

Comment:
If these are the essential attributes of this class, let's add to initializer. And what's different between `param_list` and `parameters`; `return_value` and `return_object`?